src/test/java/com/thomashan/coup/PlayerBuilder.java

```diff
@@ -3,43 +3,73 @@
 import static com.thomashan.coup.Card.AMBASSADOR;
 
 public class PlayerBuilder {
-    public static Player build(boolean active) {
-        Player player = build();
-        if (active) {
-            return player;
-        }
+    private int coins = 2;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Field coins has the same name as a method](https://app.codacy.com/app/thomashan/coup-java/pullRequest?prid=4561385)